README (2).md

```diff
@@ -0,0 +1,49 @@
+![PollCord_Logo](https://user-images.githubusercontent.com/36363608/193755078-ac3949c0-0db9-4697-bf99-cc62e3788621.png)
+
+
+[![MIT License](https://img.shields.io/badge/License-MIT-green.svg)](https://choosealicense.com/licenses/mit/)
+
+[![Python](https://img.shields.io/badge/python-v3.7+-green.svg)](https://docs.python.org/3/)
+#    PollCord
+
+### What is PollCord?
+Pollcord is a software created for Disocrd users to conduct Polls
+
+### Why PollCord?
+Disocrd is a social platform where users interact with each other. Creating Polls is a sophisticated way to ask questions, collect opinions especially with large number of users. However, Disocrd currently lacks this feature to create user-friendly interactive Polls. PollCord is an attempt to bridge this gap
+
+
+## Deployment
+
+To deploy this project run
+
+```bash
+  npm run deploy
+```
+
+
+## Documentation
+
+[Documentation](https://github.com/ntgomes/PollCord/tree/main/docs)
+
+
+## Demo
+
+Insert gif or link to demo
+
+
+## Features
+
+- Easy & User Friendly Interface to create Polls
+- Create Multiple Polls
+- Collaborator Vs Creator Only Permissions
+- Upper limit of upto 25 options for each poll 
+- Recalling Poll results
```

Comment:
This is good so far, but make sure to add descriptions to each of the features, with maybe a GIF or image for each one.